axis-ptz/camera.py

```diff
@@ -401,15 +402,23 @@ def calculateCameraPositionB(
     # Assign position and velocity of the aircraft
     a_varphi = currentPlane["lat"]  # [deg]
     a_lambda = currentPlane["lon"]  # [deg]
-    # currentPlane["latLonTime"]
+    a_time = currentPlane["latLonTime"]  # [s]
     a_h = currentPlane["altitude"]  # [m]
-    # currentPlane["altitudeTime"]
+    # currentPlane["altitudeTime"]  # Expect altitudeTime to equal latLonTime
```

Comment:
@ralatsdc - the altitudeTime is not equal to the latLonTime.  They come in as separate ADS-B messages and can add up to some sizeable error if messages get missed.  Can you split these out appropriately?